lib/shiika_parser/src/expression_parser.rs

```diff
@@ -862,11 +862,11 @@ impl<'a> Parser<'a> {
                             args.add_unnamed(e);
                         }
                     }
-                    self.skip_ws()?;
+                    self.skip_or_error(vec![Token::Space, Token::Newline], vec![Token::Semicolon])?;
                     if !self.consume(Token::Comma)? {
                         break;
                     }
-                    self.skip_wsn()?;
+                    self.skip_or_error(vec![Token::Space, Token::Newline], vec![Token::Semicolon])?;
```

Comment:
`self.skip_or_error(vec![Token::Space, Token::Newline], vec![Token::Semicolon])?;` should be function but skip_wsn function is already defined, so I can not make precise name for this.

So I left this function call as it is.